### PR TITLE
default.xml for v4.6.1 release

### PR DESCRIPTION
**Pull request by hilmarm**

### CODE REVIEW

**Review by KanjiMonster — CHANGES_REQUESTED** [review]
You also missed the tibit snippet.

File: default.xml [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-  <project name="bisdn/bisdn-onie-additions.git" path="poky/bisdn-onie-additions" remote="github" revision="master"/>
-  <project name="bisdn/meta-ofdpa.git" path="poky/meta-ofdpa" remote="github" revision="master"/>
-  <project name="bisdn/meta-open-network-linux.git" path="poky/meta-open-network-linux" remote="github" revision="master"/>
-  <project name="bisdn/meta-switch.git" path="poky/meta-switch" remote="github" revision="master"/>
+  <remote fetch="https://github.com/" name="github"/>
+  <remote fetch="ssh://git@gitlab.bisdn.de/" name="gitlab"/>
+  <remote fetch="git://git.openembedded.org/" name="oe"/>
+  <remote fetch="git://git.yoctoproject.org/" name="yocto"/>
+  
+  <project dest-branch="v4.6.1" name="bisdn/bisdn-linux.git" path="poky/build-bisdn-linux" remote="github" revision="d8eefbe3f5d86d6e7d89115e41e2529cdbe9fd85" upstream="v4.6.1"/>
```
[the remark this answers is not in the document]
Comment (hilmarm) [reply default.xml]: Added tibit snippet, and updated the revision for bisdn-linux.
Commits should be signed correctly now.

**Review by KanjiMonster — CHANGES_REQUESTED** [review]
Please squash the commits.

And your author and Signed-off-by name is still wrong:

> Signed-off-by: **hilmar.magnusson** <hilmar.magnusson@bisdn.de>

**Comment by hilmarm** [comment]
> Also Please fix your Signed-off-by and author name.

git --name fixed and commits squashed

**Review by KanjiMonster — CHANGES_REQUESTED** [review]
The author is still wrong:

> From: "hilmar.magnusson" <hilmar.magnusson@bisdn.de>

**Comment by KanjiMonster** [comment]
> The author is still wrong:
> 
> > From: "hilmar.magnusson" [hilmar.magnusson@bisdn.de](mailto:hilmar.magnusson@bisdn.de)

You can fix that with git commit --amend --author="..."

**Comment by hilmarm** [comment]
> > The author is still wrong:
> > > From: "hilmar.magnusson" [hilmar.magnusson@bisdn.de](mailto:hilmar.magnusson@bisdn.de)
> 
> You can fix that with git commit --amend --author="..."

Authorship of commit has been updated to match the one in 'signed-off-by'. Hope this is correct now

**Comment by KanjiMonster** [comment]
> > > The author is still wrong:
> > > > From: "hilmar.magnusson" [hilmar.magnusson@bisdn.de](mailto:hilmar.magnusson@bisdn.de)
> > 
> > 
> > You can fix that with git commit --amend --author="..."
> 
> Authorship of commit has been updated to match the one in 'signed-off-by'. Hope this is correct now

Looks good now.

**Review by KanjiMonster — APPROVED** [review]

**Pull request merged**